pkg/apis/k0s.k0sproject.io/v1beta1/clusterconfig_types.go

```diff
@@ -277,45 +282,50 @@ func DefaultClusterSpec(defaultStorage ...*StorageSpec) *ClusterSpec {
 	}
 }
 
-func (c *ControllerManagerSpec) Validate() []error {
-	return nil
+// Validateable interface to ensure that all config components implement Validate function
+// +k8s:deepcopy-gen=false
+type Validateable interface {
+	Validate() []error
 }
 
-var _ Validateable = (*SchedulerSpec)(nil)
```

Comment:
Why is it removed?

Reply:
I suppose this is a remnant from previous undone edits. SchedulerSpec's Validate method is a no-op, and I probably thought of removing it completely, but then decided against it for consistency. Will add it back.

Reply:
Done